tests/cql/CqlDateTimeOperatorsTest.xml

```diff
@@ -440,7 +441,7 @@
 		</test>
 		<test name="DateTimeDifferenceWeeks3">
 			<expression>difference in weeks between @2012-03-10T22:05:09 and @2012-03-24T07:19:33</expression>
-			<output>1</output>
+			<output>2</output>
```

Comment:
The spec defines Sunday as the start of the week -- so it is the boundary we should use for _difference_.  There are two Sat->Sun boundary crossovers between Mar 10 and Mar 24, so the correct difference in weeks ought to be 2.